runtime/expr/ztests/vector-spread-error.yaml

```diff
@@ -0,0 +1,8 @@
+zed: yield [...a,...b], |[...a,...b]|
+
+input: |
+  {a:[1,2],b:"hi"}
+
+output: |
+  error({message:"spread syntax requires array/set",on:"hi"})
+  error({message:"spread syntax requires array/set",on:"hi"})
```

Comment:
Nit: Merge this file into `vector-spread.yaml`.

Reply:
ok